WIP: Python API example usage for scikit model

sudharsana-kjl opened the pull request:
fix #384

pdxjohnny commented:
I think that the usage of `requests` might be better suited under a page like the javascript one (just added in #418), but for Python, under the HTTP API docs. Issue #384 was meant to be about modifying the quickstart.py code to use respective models within each model docstring. This is great for a Python HTTP API example, could you make this similar to the `service/http/docs/javascript.rst` but as `python.rst`?

sudharsana-kjl commented:
@pdxjohnny I've modified `model/scikit/dffml_model_scikit/__init__.py` and added examples for LinearRegression and Kmeans. I moved the example using requests to python.rst as suggested. Could you please review it?
